neo/public/stylesheets/main.css

```diff
@@ -114,4 +114,41 @@ search-bar {
     position: relative;
     left: 110%;
 
-}
\ No newline at end of file
+}
+
+#homepage{
```

Comment:
indeed, we should rename that id to a better name that specifically point to the `powerby` element.